components/pipeline-service/production/base/main-pipeline-service-configuration.yaml

```diff
@@ -1566,6 +1566,18 @@ spec:
                     requests:
                       cpu: 200m
                       memory: 200Mi
+        tekton-pipelines-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: tekton-pipelines-controller
+                    resources:
+                      requests:
+                        memory: 6Gi
+                        cpu: 500m
```

Comment:
Is that value enough? I see on the rh01 cluster it's going above 500m (530m) and maybe we need to leave some extra limit.

Reply:
I changed it to 1 CPU, I saw peaks around 800mi in the last 2 weeks